src/pytorch_lightning/trainer/trainer.py

```diff
@@ -1807,10 +1807,8 @@ def model(self, model: torch.nn.Module) -> None:
     @property
     def log_dir(self) -> Optional[str]:
         if len(self.loggers) > 0:
-            if not isinstance(self.loggers[0], TensorBoardLogger):
-                dirpath = self.loggers[0].save_dir
-            else:
-                dirpath = self.loggers[0].log_dir
+            logger_ = self.loggers[0]
+            dirpath = getattr(logger_, "log_dir", getattr(logger_, "save_dir", None))
```

Comment:
Why has log_dir precedence over save_dir? It should be the other way around to be consistent with the way it was before.

Reply:
:+1: